src/realm/query_expression.hpp

```diff
@@ -2620,6 +2795,18 @@ class Columns : public Subexpr2<T> {
     }
 };
 
+template <class T>
+void Columns<T>::evaluate(size_t index, ValueBase& destination)
```

Comment:
Why bother moving this out of line? It doesn't seem relevant to this PR, and it's shorter than a number of other inline implementations in this file.